src/Compilers/CSharp/Test/Semantic/Semantics/CollectionLiteralTests.cs

```diff
@@ -4584,7 +4697,6 @@ static void Main()
                 }
                 """;
             var comp = CreateCompilation(source);
-            // PROTOTYPE: Should spread elements support target type? (Should we infer IEnumerable<int>?)
```

Comment:
>PROTOTYPE [](http://example.com/codeflow?start=15&length=9)

Added open issue to test plan [https://github.com/dotnet/roslyn/issues/66418](https://github.com/dotnet/roslyn/issues/66418).